src/parser/expression.js

```diff
@@ -594,6 +598,16 @@ export default class ExpressionParser extends LValParser {
     return this.finishNode(node, "MetaProperty");
   }
 
+  parseImportMetaProperty(): N.MetaProperty {
+    const node = this.startNode();
+    const id = this.parseIdentifier(true);
+    this.expect(tt.dot);
+    if (this.options.sourceType !== "module") {
+      this.raise(id.start, "import.meta can only be used in modules");
+    }
+    return this.parseMetaProperty(node, id, "meta");
```

Comment:
Oops, good point. Added a test that checks for that & made it use the correct name.